lane.py

```diff
@@ -1997,6 +1997,46 @@ def _featured_works_with_lanes(
                 yield work, lane
 
 
+class JackpotWorkList(WorkList):
+    def initialize(self, library, filter_list):
+        super(JackpotWorkList, self).initialize(library)
+        self.library = library
+        self.filter_list = filter_list
+
+    def groups(self, _db, include_sublanes=True, facets=None,
+               search_engine=None, debug=False, **response_kwargs):
+        """
+        :param facets: A FeaturedFacets object that may restrict the works on view.
+        :param search_engine: An ExternalSearchIndex to use when
+            asking for the featured works in a given WorkList.
+        :param debug: A debug argument passed into `search_engine` when
+            running the search.
+        :yield: A sequence of (Work, WorkList) 2-tuples, with each
+            WorkList representing the child WorkList in which the Work is
+            found.
+        """
+        pagination = Pagination(size=5)
+        for (collection, medium, availability) in self.filter_list:
+            # Set up getting the filter to use for the es query.
+            from external_search import Filter
+            facets = Facets.default(self.library, availability=availability)
+            filter = Filter.from_worklist(_db, self, facets)
```

Comment:
I don't think this is a reason to delay the merge any longer, but you're taking in a `facets` argument that's being ignored. I think eventually we might want to use the `facets` argument but for now can you add a comment to the docstring that says the value is ignored?

If you want to try actually using `facets`, you can do something like `these_facets = facets.navigate(availability=availability)` inside the for loop; I think that should work.